src/patterns/data/default/entity_attribute_location.tsv

```diff
@@ -5441,3 +5441,14 @@ OBA:2045401	level of aspartate aminotransferase, mitochondrial type	PR:000008154
 OBA:2045402	creatinine level	CHEBI:16737	creatinine	PATO:0000070	amount	UBERON:0001062	anatomical entity					https://orcid.org/0000-0001-8314-2140	
 OBA:2045403	urobilinogen level	CHEBI:29026	urobilinogen	PATO:0000070	amount	UBERON:0001062	anatomical entity					https://orcid.org/0000-0001-8314-2140	
 OBA:2045404	nitrite level	CHEBI:16301	nitrite	PATO:0000070	amount	UBERON:0001062	anatomical entity					https://orcid.org/0000-0001-8314-2140	
+OBA:2045405		CHEBI:60579	calcium oxalate	PATO:0000070	amount	UBERON:0001088	urine					https://orcid.org/0000-0001-8314-2140	
+OBA:2045440	hepatic iron amount	CHEBI:18248	iron atom	PATO:0000070	amount	UBERON:0002107	liver	liver iron amount				https://orcid.org/0000-0001-8314-2140	
+OBA:2045442	arterial blood oxygen level	CHEBI:25805	oxygen atom	PATO:0000070	amount	UBERON:0013755	arterial blood					https://orcid.org/0000-0001-8314-2140	
+OBA:2045443	oxygen concentration in blood	CHEBI:25805	oxygen atom	PATO:0001159	concentrated	UBERON:0000178	blood					https://orcid.org/0000-0001-8314-2140	
```

Comment:
Why use the PATO quality `concentrated` and not `concentration of `[PATO:0000033](https://www.ebi.ac.uk/ols4/ontologies/pato/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FPATO_0000033?lang=en)?